Tests/AutoKeyboardNavigatorTests.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  AutoKeyboardNavigatorTests.swift
+//  KeyboardSupport-iOSTests
+//
+//  Created by John Davis on 12/18/18.
+//  Copyright © 2018 Bottle Rocket. All rights reserved.
+//
+
+import XCTest
+@testable import KeyboardSupport
+
+class AutoKeyboardNavigatorTests: XCTestCase {
+
+    // MARK: - Properties
+    
+    private var keyboardNavigator: KeyboardAutoNavigator?
+    private var delegateMock: MockKeyboardAutoNavigatorDelegate?
+    
+    // MARK: - Tests
+    
+    override func setUp() {
+        super.setUp()
+        delegateMock = MockKeyboardAutoNavigatorDelegate()
+        keyboardNavigator = KeyboardAutoNavigator(navigationContainer: UIView(), defaultToolbar: KeyboardToolbar(), returnKeyNavigationEnabled: true)
+        keyboardNavigator?.delegate = delegateMock
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        keyboardNavigator = nil
+    }
+
+    func test_keyboardAutoNavigator_invokesDelegateOnNext() {
+        keyboardNavigator?.didTapNext()
```

Comment:
Just to follow up on a previous comment about the redundancy of `didTapNext()`, `didTapBack()`, and `didTapDone()`, we would invoke the `KeyboardAccessoryDelegate` methods instead. So this might become:
```swift
    func test_keyboardAutoNavigator_invokesDelegateOnNext() {
        keyboardNavigator?.keyboardAccessoryDidTapNext(UIView())
        XCTAssertEqual(delegateMock?.keyboardNavigatorDidTapNextCount, 1)
    }
```

Reply:
Fixed in the helperMethod public-private change.